### ISSUE TITLE
Local Authorizer only called on first invocation

### ISSUE THREAD

**Issue by connorgiles** [nlang/serverless-offline-local-authorizers-plugin#8]
Thanks for the package! Having an issue where the local authorizer is only called once. The first time hitting the endpoint, the authorizer works as expected. After that, the authorizer is not called on subsequent requests and results in an authorization error.

I even attempted to use a mock authorizer with the same result:
```typescript
const mockedAuthService = async () => {
  console.log('Called auth service')
  return {
    principalId: 'test',
    policyDocument: {
      Statement: [
        {
          Action: 'execute-api:Invoke',
          Effect: 'Allow',
          Resource: '*',
        },
      ],
      Version: '2012-10-17',
    },
  }
}

module.exports = {mockedAuthService}
```
![image](https://user-images.githubusercontent.com/9627888/92296636-ad9ee200-ef04-11ea-9848-b88bc6e651ca.png)

Any idea as to what might be causing this? The rest of the service is written in typescript and compiled with webpack. Also, here is my serverless config for the endpoint in case it is relevant:

```yml
functions:
  transaction:
    handler: src/handlers/transactions.getTransaction
    events:
      - http:
          method: get
          path: /transactions/{organizationId}/{transactionId}
          authorizer:
            arn: # ARN for authorizer
            identitySource: method.request.header.Authorization
            identityValidationExpression: ^Bearer .+$
            localAuthorizer:
              name: mockedAuthService
```

**Comment by connorgiles** [comment]
I also verified and the same does not happen if I simply mock out a custom authorizer using basic `serverless-offline`. This is what makes me think it's part of the plugin. Has anyone else experienced this issue? Here is how I mocked it which resulted in the successful authorization for each request.

```typescript
export const authorize = async () => {
  console.log('Authorize')
  return {
    principalId: 'test',
    policyDocument: {
      Statement: [
        {
          Action: 'execute-api:Invoke',
          Effect: 'Allow',
          Resource: '*',
        },
      ],
      Version: '2012-10-17',
    },
  }
}

```

```yml
functions:
  authorize:
    handler: src/handlers/authorizer.authorize

  transaction:
    handler: src/handlers/transactions.getTransaction
    events:
      - http:
          method: get
          path: /transactions/{organizationId}/{transactionId}
          parameters:
            paths:
              organizationId: true
              transactionId: true
          authorizer:
            name: authorize
```

**Comment by nlang** [comment]
I tried to reproduce your error but couldn't. The repo contains a simple example application, have you tried running that and does it behave as it should on your machine?

**Comment by connorgiles** [comment]
Thanks for the quick response. I tried the example project and as you said, it worked as expected. I'm wondering if it may have anything to do with `serverless-webpack` or the fact that I'm using typescript. 

I noticed the library is written in typescript though which makes me think it has maybe been used with a TS service in the past. Have you had success using this with Webpack and Typescript? Was there anything unique about the configuration?
